dance/atlas/sc_similarity/anndata_similarity.py

```diff
@@ -0,0 +1,520 @@
+# anndata_similarity.py
```

Comment:
Calculate the similarity of data sets and return the most similar data set in the atlas for the query data set